piet-wgsl/src/render.rs

```diff
@@ -209,6 +210,7 @@ pub fn render_full(
     // TODO: calculate for real when we do rectangles
     let n_drawobj = n_path;
     let n_clip = data.n_clip;
+    let bin_data_start = n_drawobj * MAX_DRAWINFO_SIZE as u32;
```

Comment:
Now that I’ve internalized the drawtag encoding, it seems like we can compute this exact size for free in SceneBuilder. Consider this more of a note to self rather than a suggestion for this PR.